Presentation.Web/app/components/global-admin/global-admin-misc.view.html

```diff
@@ -14,8 +14,43 @@ <h3>Rapport over brudte links</h3>
         <p><strong>Antal registrerede fejl:</strong> {{brokenLinksVm.brokenLinksCount}}</p>
         <a class="btn btn-default" id="brokenLinksReportAnchor" href="{{brokenLinksVm.linkToCurrentReport}}">Hent rapporten</a>
     </div>
-    <div class="well" ng-if="!brokenLinksVm.available">
-        <p>Rapporten over brudte links er endnu ikke tilgængelig. Forventet tilgængelighed er "Mandag i næste uge".</p>
+     <div class="well" ng-if="!brokenLinksVm.available">
+         <p>Rapporten over brudte links er endnu ikke tilgængelig. Forventet tilgængelighed er "Mandag i næste uge".</p>
+     </div>
+    
+    <h3>Nedlæg bruger</h3>
+    <div class="well">
+        <div class="row" style="margin-bottom: 15px">
+            <input id="add-new-user-for-delete"
+                   class="col-xs-11"
+                   ui-select2="::userOptions"
+                   placeholder="Vælg bruger"
+                   ng-model="selectedUser">
+            <button class="btn btn-danger btn-sm col-xs-1"
+                    ng-disabled="!userSelected"
```

Comment:
you should have binded to `!selectedUser` since that is also used by the selector control and if undefined it will be the same as this.. then you don't have to maintain yet another variable in the controller